### ISSUE TITLE
2.7.0: ClassNotFoundException: org.distributeme.core.stats.RoutingStats

### ISSUE THREAD

**Issue by fenders** [anotheria/moskito#91]
Hi, I recently upgraded our application to MoSKito 2.7.0. Doing so, I ran into several ClassNotFoundExceptions referring to org.distributeme.core.stats.RoutingStats.
I found that this class is included in distributeme-core-2.2.4 but not in distributeme-core-2.2.2, which is included in moskito-inspect-standalone:2.7.0 from Maven Central.
Replacing the distributeme libraries within the war solved the issue.

**Comment by dvayanu** [comment]
Hello,

do you use both moskito and distributeme in your application, or do you only use moskito. 2.7.0 should actually grab the proper version of distributeme, and it would be an error if it doesn't do it by itself.

**Comment by fenders** [comment]
Hi, thanks for your quick response.

I don't use distributeme explicitly. I am using moskito-inspect-remote alongside moskito-inspect-standalone:2.7.0 (as war file within the same container).
When upgrading moskito-inspect-remote and moskito-inspect-standalone to 2.7.0, I encountered various ClassNotFoundExceptions in our tomcat log. 
The moskito-inspect-standalone war file, which I downloaded from Maven, contains the distributeme libraries in version 2.2.2, which do not contain the desired class.
The error messages vanished, when I manually replaced the distributeme libraries within the moskito-inspect-standalone war file with version 2.2.4.

All the best
Frank
